### PR TITLE
Following list example / demo script

### PR DESCRIPTION
**Pull request by cyle**
This adds a simple example / demo script, and an `examples` folder, to help describe how to use this library. It's an alternative way of looking at the list of what blogs you follow.

cc @sanmai

### CODE REVIEW

File: examples/following-list/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+... and now let's set up the Tumblr API client itself, so we can start asking Tumblr for information. After the above four lines, add these:
+
+```php
+$consumer_key = ''; // put your consumer key between the ‘'
+$consumer_secret = ''; // put your consumer secret between the ‘'
+$token_key = ''; // put your token between the ‘'
+$token_secret = ''; // put your token secret between the ‘'
+
+$tumblr = new \Tumblr\API\Client($consumer_key, $consumer_secret, $token_key, $token_secret);
```
Comment (sanmai) [inline examples/following-list/README.md:42]: I supposed we need to update our examples to remove any confusion.

https://api.tumblr.com/console/calls/user/info

**Review by sanmai — APPROVED** [review]
Followed the steps, got a working example!

File: examples/following-list/README.md (hunk 1 above)
Comment (cyle) [reply examples/following-list/README.md:42]: what do you think we need to update? i'm not sure i follow

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,5 @@
 /phpunit.xml
 /build/
 /.*.cache
+/.idea/
```
[the remark this answers is not in the document]
Comment (kgnfth) [reply .gitignore:6]: Ok you do that.

**Pull request merged**